instrumentation/github.com/bradfitz/gomemcache/gomemcache.go

```diff
@@ -0,0 +1,230 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gomemcache
+
+import (
+	"context"
+
+	"github.com/bradfitz/gomemcache/memcache"
+
+	otelglobal "go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/standard"
+	oteltrace "go.opentelemetry.io/otel/api/trace"
+)
+
+const (
+	defaultTracerName = "go.opentelemetry.io/contrib/instrumentation/github.com/bradfitz/gomemcache"
+	tracerVersion     = "semver:0.1.0"
```

Comment:
nice :+1: 

This is a great example pattern for other instrumentation to follow in https://github.com/open-telemetry/opentelemetry-go-contrib/issues/214. Thanks for the reference.